models/tests/layers/block/test_block_graphconv.py

```diff
@@ -63,7 +67,7 @@ class TestGraphConvMapperBlock:
         in_channels=st.integers(min_value=1, max_value=100),
         out_channels=st.integers(min_value=1, max_value=100),
         mlp_extra_layers=st.integers(min_value=1, max_value=5),
-        activation=st.sampled_from(["ReLU", "GELU", "Tanh"]),
+        activation=st.sampled_from(["torch.nn.ReLU", "torch.nn.GELU", "anemoi.models.layers.activations.GLU"]),
         update_src_nodes=st.booleans(),
```

Comment:
Since `SwiGLU` is now stable should we add it to the test to ensure it doesn't break in the future?

Reply:
Should I include it here as well? It is tested as part of the GLUs in the other tests, so by extension _should_ work here. Just didn't want to blow up the functions.